Lesson8/client.py

```diff
@@ -0,0 +1,192 @@
+import sys
```

Comment:
There're numerous PEP8 violations:
![image](https://user-images.githubusercontent.com/2947151/89710079-b7c2d480-d988-11ea-9bc4-ac375518089f.png)

Reply:
I fixed it